charts/rpe-feature-toggle-api/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart for the feature-toggle-api (https://github.com/hmcts/feature-toggle-api)
 name: rpe-feature-toggle-api
-version: 3.0.1
+version: 3.0.2
```

Comment:
also need to bump:
https://github.com/hmcts/feature-toggle-api/blob/master/charts/rpe-feature-toggle-api/requirements.yaml
and change to hmctspublic